scss/_reboot.scss

```diff
@@ -205,6 +205,11 @@ a {
     color: $link-hover-color;
     text-decoration: $link-hover-decoration;
   }
+
+  // Reset the `cursor` on non-disabled links. Also applies to buttons.
+  &:not(:disabled):not(.disabled) {
+    cursor: pointer;
```

Comment:
this would also force `cursor:pointer` for any anchors (`<a name="blah">`, or more generally `<a>` elements without an `href`). this will likely make react people happy (as they're usually the ones complaining, if memory serves), but will end up with confusing experience on sites that do use static anchors that are not links?

Reply:
solution if we did want to just do this for real links would be to change selector to something like `&[href]:not(disabled):not(.disabled)`

Reply:
Is this code necessary? When `<a>` is a link, browser makes the cursor to pointer by default.

Reply:
ah, good point. yes, it seems at that point, it's redundant (and nicely circumvents the whole "what about anchors" dilemma)